modules/xmpp/SignalingLayerImpl.js

```diff
@@ -52,6 +52,16 @@ export default class SignalingLayerImpl extends SignalingLayer {
          * @private
          */
         this._remoteSourceState = { };
+
+        /**
+         * A map that stores the source name of a track identified by it's ssrc.
+         * We store the mapping when jingle is received, and later is used
+         * onaddstream webrtc event where we have only the ssrc
+         * FIXME: This map got filled and never cleaned and can grow during long
```

Comment:
can you try cleaning it up here:
https://github.com/jitsi/lib-jitsi-meet/blob/6bff32f86e0615e78c5a2aca9f64f8bc08a8a0f9/modules/xmpp/SignalingLayerImpl.js#L246
maybe it's possible to get SSRC owner and then remove from the map based on that?

Reply:
ok will take a look

Reply:
updated code here.  please check new logic.  it iterates through the ssrcOwners map to find all relevant ssrc to delete based on endpointId.  this works right now only because ssrcOwners is not cleaned up, and its a full iteration each time and ssrcOwners grows indefinitely atm.  if ssrcOwners is cleaned up in a different PR, then care needs to be taken here as the ordering of the cleanup would matter.

Reply:
great, thanks!